packages/babel-parser/src/plugins/typescript/index.ts

```diff
@@ -90,12 +90,14 @@ const TSErrors = ParseErrorEnum`typescript`({
     propertyName: string;
   }) =>
     `Property '${propertyName}' cannot have an initializer because it is marked abstract.`,
-  AccesorCannotDeclareThisParameter:
+  AccessorCannotBeOptional:
```

Comment:
We can (and should) fix the error messages now (i.e. `BadSetterArity`/`BetGetterArity`), but we should wait until Babel 8 to fix typos in error _codes_.

Reply:
Do error codes refer to the property names? These seem to be only visible from the inside.

Reply:
Yes they are the property names; and they are public. For example, prettier relies on them: https://github.com/prettier/prettier/blob/b50dfd1974d33abb980e9b4df949913932eeb478/src/language-js/parse/babel.js#L183